{{cookiecutter.project_name}}/.github/workflows/main.yml

```
@@ -53,6 +53,8 @@ jobs:
       - name: Upload coverage reports to Codecov with GitHub Action on Python 3.11
         uses: codecov/codecov-action@v4
         if: {% raw %}${{ matrix.python-version == '3.11' }}{% endraw %}
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Since this is in a template, we need to add;
```suggestion
         {% raw %}CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}{% endraw %}
```